users/details/style.css

```diff
@@ -20,6 +20,7 @@ html {
   --blue-hover-color: #0d0a4d;
   --razzmatazz-color: #e30464;
   --razzmatazz-hover-color: #a1034a;
+  --gray-color: gray;
 }
```

Comment:
color hex code please, please follow the pattern of the code base

Reply:
It's not necessary. That's why we are using variables